### PR TITLE
WallUnitTest update for interzonal airflow v2

### PR DESCRIPTION
**Pull request by Mathadon**
@jelgerjansen this is an update to `WallUnitTest` model.

### CODE REVIEW

**Comment by jelgerjansen** [comment]
@Mathadon can you add a newline at the end of both files (I cannot push to your branch)? After this I will merge the changes.

File: IDEAS/Buildings/Components/Examples/WallUnitTest.mo [hunk 1]
```diff
@@ -98,4 +101,4 @@ First implementation.
 </li>
 </ul>
 </html>"));
-end WallUnitTest;
+end WallUnitTest;
```
Comment (jelgerjansen) [inline IDEAS/Buildings/Components/Examples/WallUnitTest.mo]: Can you add a newline at the end of the file? Furthermore, can you update the revision section.

File: IDEAS/Buildings/Components/Interfaces/DummyConnection.mo [hunk 2]
```diff
@@ -164,4 +168,4 @@ See <a href=\"https://github.com/open-ideas/IDEAS/issues/1131\">
 </li>
 </ul>
 </html>"));
-end DummyConnection;
+end DummyConnection;
```
Comment (jelgerjansen) [inline IDEAS/Buildings/Components/Interfaces/DummyConnection.mo]: Can you add a newline?

File: IDEAS/Buildings/Components/Interfaces/DummyConnection.mo [hunk 3]
```diff
@@ -132,17 +135,18 @@ equation
...
+  connect(hZone.y, zoneBus.hfloor) annotation(
+    Line(points = {{-20, 74}, {100, 74}, {100, -2}}, color = {0, 0, 127}));
+  connect(bou.ports[2], zoneBus.port_2) annotation(
+    Line(points = {{-20, 40}, {100, 40}, {100, -2}}, color = {255, 204, 51}));
+  connect(bou.ports[1], zoneBus.port_1) annotation(
+    Line(points = {{-20, 40}, {100, 40}, {100, -2}}, color = {255, 204, 51}));
+   annotation(
+    Diagram(coordinateSystem(preserveAspectRatio = false, extent = {{-100, -100}, {100, 100}})),
+    Icon(coordinateSystem(preserveAspectRatio = false, extent = {{-100, -100}, {100, 100}}), graphics = {Line(points = {{60, 60}, {-60, -60}}, color = {0, 0, 255}), Line(points = {{-60, 60}, {60, -60}}, color = {0, 0, 255})}),
+    Documentation(revisions = "<html>
```
Comment (jelgerjansen) [inline IDEAS/Buildings/Components/Interfaces/DummyConnection.mo:149]: Can you update the revision section?

File: IDEAS/Buildings/Components/Examples/WallUnitTest.mo [hunk 4]
```diff
@@ -99,3 +102,4 @@ First implementation.
 </ul>
 </html>"));
 end WallUnitTest;
+
```
Comment (jelgerjansen) [inline IDEAS/Buildings/Components/Examples/WallUnitTest.mo]: Can you update the revision section?

File: IDEAS/Buildings/Components/Interfaces/DummyConnection.mo [hunk 5]
```diff
@@ -12,12 +14,11 @@ model DummyConnection "Source generator/sink for propsbus"
...
-  outer IDEAS.BoundaryConditions.SimInfoManager sim
-    annotation (Placement(transformation(extent={{-100,20},{-80,40}})));
+  outer IDEAS.BoundaryConditions.SimInfoManager sim annotation (Placement(transformation(extent={{-100,20},{-80,40}})));
   IDEAS.Buildings.Components.Interfaces.ZoneBus zoneBus(
-    redeclare package Medium = IDEAS.Media.Air,
-    outputAngles=sim.outputAngles,
-    numIncAndAziInBus=sim.numIncAndAziInBus)
+    redeclare package Medium = Medium,
+  numIncAndAziInBus=sim.numIncAndAziInBus,
+    outputAngles=sim.outputAngles, use_port_1 = sim.interZonalAirFlowType <> IDEAS.BoundaryConditions.Types.InterZonalAirFlow.None, use_port_2 = sim.interZonalAirFlowType == IDEAS.BoundaryConditions.Types.InterZonalAirFlow.TwoPorts)
```
Comment (jelgerjansen) [inline IDEAS/Buildings/Components/Interfaces/DummyConnection.mo:21]: Could you put `use_port_1` and `use_port_2` on a new line (to improve readability).

**Comment by Mathadon** [comment]
@jelgerjansen all done!

**Pull request merged**